src/js/modules/infragistics.ui.editors.js

```diff
@@ -4499,11 +4509,30 @@
 			if (this.options.scientificFormat) {
 				this.options.spinDelta = Number(this.options.spinDelta.toExponential());
 			}
-			if (this.options.maxLength !== null) {
-				this.options.maxLength = null;
+		},
+		_validateDecimalSettings: function() {
+			this._validateDecimalSetting("minDecimals", this.options.minDecimals);
+			this._validateDecimalSetting("maxDecimals", this.options.maxDecimals);
+			this._validateDecimalMinMax();
+		},
+		_validateDecimalSetting: function(name, value) {
+			var mode = this.options.dataMode, boundary;
+
+			if (mode === "double") {
+				boundary = 15;
+			} else if (mode === "float") {
+				boundary = 7;
 			}
-			if (this.options.value < 0) {
-				this._editorInput.addClass(this.css.negative);
+
+			if (value === "" || isNaN(value) ||
+				(!isNaN(value) && (value < 0 || value > boundary))) {
+				throw new Error($.ig.util.stringFormat($.ig.Editor.locale.decimalNumber,
```

Comment:
@damyanpetev  I will add a new issue to throw the exceptions at initialization time (i.e. break the editors) in case we have such configuration errors. 
//cc @kdinev